docs/windows.md

```diff
@@ -59,7 +59,39 @@ and daemons are pre-installed on the Windows Nodes in the demo.
     See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes)
     section.
 
-### Installation
+### Installation as a Service (containerd based runtimes)
+
+First install antrea (0.13+ is required for containerd).
+
+```bash
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.13.0/antrea.yml
+```
+
+Then, you can run the following commands.  This will setup antrea as a windows service.
+```
+choco install yq
+curl.exe -LO https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Helper.psm1 
+curl.exe -LO https://github.com/vmware-tanzu/antrea/releases/download/v0.13.0/antrea-agent-windows-x86_64.exe
+
+mv antrea-agent-windows-x86_64.exe c:/k/antrea/bin/antrea-agent.exe
+Import-Module ./helper.psm1
+Get-
```

Comment:
The formatting looks strange here: why is this on its own line?

Reply:
Removed.